api-platform/core/3.2/config/packages/api_platform.yaml

```
@@ -0,0 +1,26 @@
+api_platform:
+    title: Hello API Platform
+    version: 1.0.0
+    formats:
+        jsonld:
+            - application/ld+json
+    doc_formats:
+        jsonopenapi:
+            - application/vnd.openapi+json
+        jsonld:
+            - application/ld+json
+      html:
+        - text/html
+    event_listeners_backward_compatibility_layer: false
+    keep_legacy_inflector: false
+    defaults:
+        stateless: true
+        cache_headers:
+            vary:
+                - Content-Type
+                - Authorization
+                - Origin
```

```suggestion
    formats:
        jsonld: ['application/ld+json']
    doc_formats:
        jsonld: ['application/ld+json']
        html: ['text/html']
    event_listeners_backward_compatibility_layer: false
    keep_legacy_inflector: false
    defaults:
        stateless: true
        cache_headers:
            vary: ['Content-Type', 'Authorization', 'Origin']
```

(JSON:API docs format shouldn't be enabled by default)

---

and openapi in its JSON format?

---

Sorry my bad, indeed `application/vnd.openapi+json` should be enabled. I read too quickly and confused it with JSON:API.